rezervo/api/user.py

```diff
@@ -0,0 +1,39 @@
+from auth0.management import Auth0
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+from starlette import status
+
+from rezervo.api.common import get_db, token_auth_scheme
+from rezervo.auth import auth0
+from rezervo.auth.auth0 import get_auth0_management_client
+from rezervo.database import crud
+from rezervo.settings import Settings, get_settings
+
+router = APIRouter()
+
+
+@router.get("/user")
```

Comment:
Why change it to `GET`? I think both `POST` and `PUT` would fit better here, mostly because the endpoint is actually performing an update operation. A `GET` without a response body feels a little odd :thinking:

Reply:
I mean, it is either a `GET` without any response or a `POST/PUT` without any data. Buy sure, we can keep the PUT since that mostly approximates what the endpoint is actually doing 🤠 ✅

Reply:
Probably none of them would be proper RESTful 🙃, but I think it's important to signal that this is modifying data and not just requesting a resource. `PUT` seems good for an upsert operation :+1: (an empty body is not great since it could imply clearing the user resource, but I think the alternative of hiding a mutation is worse)